src/main/java/net/snowflake/ingest/streaming/internal/EpInfo.java

```diff
@@ -17,6 +21,30 @@ class EpInfo {
     this.columnEps = columnEps;
   }
 
+  /** Some basic verification logic to make sure the EP info is correct */
+  public void verifyEpInfo() {
```

Comment:
FYI we have this check server side now as well. We could always remove it here if you want?

Reply:
This is before the file is built and customer can see it right away so they can save the log and report it, it doesn't hurt to verify it twice but i feel like adding the check at server side is too late.